micro_http/src/common/mod.rs

```diff
@@ -28,6 +28,17 @@ pub enum RequestError {
     InvalidRequest,
 }
 
+/// Errors associated with a HTTP Connection.
+#[derive(Debug, PartialEq)]
+pub enum ConnectionError {
+    /// The request parsing has failed.
+    ParseError(RequestError),
+    /// Could not perform a stream operation successfully.
+    StreamError,
+    /// Attempted to read or write on a closed connection.
+    ClosedConnection,
```

Comment:
`ConnectionClosed` is typically used and not `ClosedConnection`. Can you also update this one?